crypto-keys/src/main/java/com/palantir/crypto2/keys/serialization/KeyMaterials.java

```diff
@@ -78,18 +81,37 @@ public static KeyMaterial from(String keyAlgorithm, byte[] encodedKey, byte[] iv
         return KeySerializerV2.INSTANCE.wrap(keyMaterial, key);
     }
 
+    /**
+     * See {@link SymmetricKeySerializer} to understand when it is acceptable to use symmetric key wrapping.
+     */
+    public static byte[] symmetricWrap(KeyMaterial keyMaterial, SecretKey key) {
```

Comment:
maybe call this one `wrap`, too? the arguments should differentiate between the two